regression/cbmc/CMakeLists.txt

```diff
@@ -1,20 +1,45 @@
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
+add_test_pl_tests(
+    "$<TARGET_FILE:cbmc> --validate-goto-model" -X smt-backend -X gcc-only
+)
+else()
 add_test_pl_tests(
     "$<TARGET_FILE:cbmc> --validate-goto-model" -X smt-backend
 )
+endif()
 
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
```

Comment:
use `set(...)` and `string(CONCAT ...)` to avoid duplication, similar to Makefile case

Reply:
Thanks!